mne/encoding/model.py

```diff
@@ -0,0 +1,329 @@
+import numpy as np
+from sklearn.linear_model import Ridge
+from sklearn.pipeline import Pipeline
+from .feature import EventsBinarizer, DataDelayer
+from ..io.pick import pick_types, pick_info
+from ..externals.six import string_types
+from ..utils import warn
+from ..evoked import EvokedArray
+from copy import deepcopy
+
+
+class EventRelatedRegressor(object):
+    """Fit an rER[R/F] model to Raw objects.
+
```

Comment:
As in, don't import `EncodingModel` to `mne.encoding`, only the higher-level objects built on top of it? I think that'd fine. Maybe in that case we could also build a `ReceptiveFieldModel` that would fit continuous data and have a method like `to_receptive_field` instead of the `to_evoked` that `EventRelatedRegressor` has. WDYT?